minecode/tests/collectors/test_maven.py

```diff
@@ -500,12 +500,262 @@ def test_get_artifact_sha1(self, mock_request_get):
 
     def test_get_classifier_from_artifact_url(self):
         artifact_url = "https://repo1.maven.org/maven2/net/alchim31/livereload-jvm/0.2.0/livereload-jvm-0.2.0-onejar.jar"
-        package_version_page_url = (
-            "https://repo1.maven.org/maven2/net/alchim31/livereload-jvm/0.2.0/"
-        )
         package_name = "livereload-jvm"
         package_version = "0.2.0"
         classifier = maven.get_classifier_from_artifact_url(
-            artifact_url, package_version_page_url, package_name, package_version
+            artifact_url, package_name, package_version
         )
         self.assertEqual("onejar", classifier)
+
+    def test_collect_links_and_artifact_timestamps_repo_maven_apache_org(self):
+        # https://repo.maven.apache.org/maven2
+        text = """
```

Comment:
The html code should be in individual test files, then we use them like this https://github.com/aboutcode-org/purldb/blob/main/packagedb/tests/test_api.py#L220